src/main/java/kafka/Kafka.java

```diff
@@ -0,0 +1,51 @@
+package kafka;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+
+public class Kafka {
+
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+    private static boolean isExitChat = false;
+
+    public Kafka(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            taskList = new TaskList(storage.load());
+        } catch (FileNotFoundException e) {
+            System.out.println("File not found");
+            taskList = new TaskList();
+        }
+    }
+
+    public static void exitChat() {
+        if (!isExitChat) {
+            isExitChat = true;
+        }
+    }
+    public void run() {
+        Scanner scanner = new Scanner(System.in);
+
+        ui.greet();
+        while (!isExitChat) {
+            try {
+                Parser.parseCommand(scanner.nextLine(), taskList, storage, ui);
+            } catch (KafkaException | IOException e) {
+                ui.showError(e);
+            } catch (DateTimeParseException e) {
+                ui.incorrectDateDetails();
+            }
+        }
+        ui.goodbye();
+    }
+
+    public static void main(String[] args) {
+        new Kafka("C:/Users/Nicholas/Downloads/Kafka.txt").run();
```

Comment:
Hello, I think it might not work on other computer (even if it's windows) because the path directory won't be the same. Perhaps it's better to use a relative path instead? And maybe it's better to handle the case where a folder or a txt file has not being created yet to prevent checked exception?
![Screenshot 2024-09-07 191711](https://github.com/user-attachments/assets/e0931b04-4216-40b4-abf1-14a0354e8a81)